chapter_appendix-mathematics-for-deep-learning/naive-bayes.md

```diff
@@ -110,7 +119,7 @@ label, type(label)
 
 ```{.python .input}
 #@tab tensorflow
-label, type(label)
+label.numpy(), label.dtype
```

Comment:
Why is this needed?

Reply:
For the same reason as above. We  can ignore `label.numpy()`. But `label.dtype` is necessary. Otherwise, `type(label)` gives `tf.blah.blah.blah.EagerTensor`.